doc/bib/elephant.bib

```diff
@@ -62,4 +62,14 @@ @article{Riehle97_1950
   number = {5345},
   pages = {1950--1953},
   doi = {10.1126/science.278.5345.1950}
-}
\ No newline at end of file
+}
+
+@article{torre2016asset,
+  title={ASSET: analysis of sequences of synchronous events in massively parallel spike trains},
+  author={Torre, Emiliano and Canova, Carlos and Denker, Michael and Gerstein, George and Helias, Moritz and Gr{\"u}n, Sonja},
+  year={2016},
+  journal={PLoS computational biology},
+  volume={12},
+  number={7},
+  doi={10.1371/journal.pcbi.1004939}
```

Comment:
Missing the page: e1004939

Reply:
I'm not very comfortable with citing
should it be `pages = {e1004939}`?

Reply:
Yes. Although they do not have actual pages in PLoS, `pages = {e1004939}` corresponds to a record with only the starting page field in other formats (such RIS).